code/__DEFINES/projectile_defines.dm

```diff
@@ -18,6 +18,13 @@
 #define CAL_CROSSBOW "bolt"
 #define CAL_ARROW "arrow"
 
+//Mech Caliber Defines
+#define CAL_MECH_MACHINEGUN 	"20mm"
+#define CAL_MECH_AUTOCANNON		"50mm HEAD"
+#define CAL_MECH_ULTRACANNON	"40mm HEAD"
```

Comment:
40mm is already used in guns

Reply:
Trilby we changed .40 to 10mm long ago, where do we even use 40mm in guns, our highest bore is 14.5mm

Reply:
its used for nades

Reply:
Increased both ACs by 10mm to 60 50, for these don't really care how big they are considering in source material I used they go up to 200